src/timemanager.cxx

```diff
@@ -100,7 +110,8 @@ void TimeManager::setFpb(double f)
 	int bpm = ( samplerate * 60) / f;
 
 	char buffer [50];
-	sprintf (buffer, "TM, setFpb() %i, bpm = %i", int(f), int(bpm) );
+	snprintf(buffer, sizeof(buffer), "TM, setFpb() %i, bpm = %i",
+		  int(f), int(bpm) );
```

Comment:
better use _fpb, just becaust its better naming ;)

Reply:
`f` is the name of the variable in the function prototype, so leaving as is.

Reply:
some lines earlier there is: `_fpb = f;` so it makes no difference but fpb is more telling.